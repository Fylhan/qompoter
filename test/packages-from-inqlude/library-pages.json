{
	"name": "packages-from-inqlude/copy-offline",
	"authors": [
		{
			"name": "Fylhan",
			"homepage": "http://fylhan.la-bnbox.fr"
		}
	],
	"require": {
		"lathiat/avahi-qt": "v0.6.32",
		"kde/baloo": "v5.24.0",
		"kde/attica": "v5.25.*",
		"kde/bluez-qt": "v5.24.*"
	}
}
