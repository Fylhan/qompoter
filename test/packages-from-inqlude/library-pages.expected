Qompoter
======== install

* kde/attica v5.25.*
  Selection version: v5.25.0
  Downloading sources from Git...
  Warning: error with Git, Qompoter will try downloading sources from scratch...
  FAILURE

FAILURE
